### ISSUE TITLE
box-sizing rule in reset.css causes scrollbars across all of MDN

### ISSUE THREAD

**Issue by tombigel** [mdn/yari#3025]
I can't say since when, but lately every example in an `iframe` on MDN has a vertical scrollbar.

This is caused by a rule in `reset.scss` that forces `box-sizing: border-box` on everything (apparently ignoring existing css rules that doesn't take it into account).
Removing the rule and suddenly MDN looks good again (see attached gif of a screen recording).

![Screen Recording 2021-02-23 at 12 43 59](https://user-images.githubusercontent.com/227922/108833151-170ace00-75d5-11eb-89de-5a2c4230cea8.gif)

**Comment by schalkneethling** [comment]
Thank you for reporting @tombigel. We will have a look and decide how to best handle this situation. I would not want to remove the `box-sizing` rule. Other than those live examples, have you seen anything else affected?

**Comment by tombigel** [comment]
No... besides some layout shifts that either could be the right version.

Since I deal a lot with layouts at work, I keep my Mac with the "Always show scrollbars" setting turned on. You catch a lot of things that are hidden from other devs on Macs this way :)
